### ISSUE TITLE
Cannot render JSON

### ISSUE THREAD

**Issue by jsguy** [StephanHoyer/mithril-node-render#12]
Hey Stephan :)

in the latest version, it always escapes JSON, so if you want to render JS variables, it won't work, for example:

```javascript
{
	controller: function(params) {
		this.js = {"hello": "world"};
		return this;
	},
	view: function(ctrl) {
		return m("SCRIPT", "var who = " + JSON.stringify(ctrl.js));
	}
}
```

Should get:

```markup
<script>var who = {"hello":"world"}</script>
```

But get:

```markup
<script>var who = {&quothello&quot:&quotworld&quot}</script>
```

**Comment by StephanHoyer** [comment]
Hi,

I changed this is for security reasons. Output is now always quoted as mithril also does.

you can avoid the quoting by using 
```javascript
return m('SCRIPT', m.trust('var who = ' + JSON.stringify(ctrl.js));
```

Didn't test if this works for JSON. May you can and confirm that it's working.

**Comment by jsguy** [comment]
Yep, confirmed working, cheers! :+1:

**Comment by jsguy** [comment]
Ah, wait, no, I was still using 0.1.5

**Comment by StephanHoyer** [comment]
Stange

I'll fix that tomorrow. Fell free to open a PR if you want

**Comment by jsguy** [comment]
I've created a mithril demo here:

http://codepen.io/jsguy/pen/azGmbw?editors=101

I'm not sure how to best solve this - do we need to override m.trust, so that we know the call is from there?

**Comment by jsguy** [comment]
Also, here is a runnable, with the same view to show the issue in node: 

http://web-b7843201-9634-4b57-b4ae-2203d5cc5bbd.runnable.com

**Comment by StephanHoyer** [comment]
Just released 0.2.2

I now only escape quotes for attributes. Hope that solves the issue.

**Comment by jsguy** [comment]
Works for me, thanks! :+1:
